pkg/skuba/actions/auth/auth.go

```diff
@@ -186,7 +186,7 @@ func doAuth(authReq request) (*response, error) {
 	ctx := oidc.ClientContext(context.Background(), client)
 	provider, err := oidc.NewProvider(ctx, authReq.IssuerURL)
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to query provider %q", authReq.IssuerURL)
+		return nil, fmt.Errorf("URL %s is not a OIDC dex server (maybe missing --root-ca or --insecure, or incorrect port number?)", authReq.IssuerURL)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to query provider %s (is this the right URL? maybe missing --root-ca or --insecure, or incorrect port number?)", authReq.IssuerURL)
```

Reply:
Failure doesn't necessary always the case of `URL %s is not a OIDC dex server` and seems inconsistent with the followed suggest fix -> `maybe missing --root-ca or --insecure, or incorrect port number?` 

IMHO the original message is clear without the wrapped up errors.